exo-go/src/types/docker_config.go

```diff
@@ -4,7 +4,7 @@ package types
 // docker-compose.yml
 type DockerConfig struct {
 	Image         string            `yaml:",omitempty"`
-	Build         string            `yaml:",omitempty"`
+	Build         map[string]string `yaml:",omitempty"`
```

Comment:
Yes that's what I meant. Set `build` to `../tests` for `exo tests` and `./service` for `exo run`. The user should have to copy `package.json` from its parent directory and then run install. (It shouldn't be running `npm install` like that, see: https://github.com/Originate/space-tweet/blob/master/users-service/Dockerfile#L4). But regardless, the test Dockerfile will live under `service/tests` and reference files in `service/`, so its build commands should reflect that.